### ISSUE TITLE
0.0 in content behaves weirdly

### ISSUE THREAD

**Issue by tv42** [typst/typst#179]
```shell
$ echo 0.0 >foo.typ
$ typst foo.typ
error: must be positive
  ┌─ foo.typ:1:0
  │
1 │ 0.0
  │ ^^
```

**Comment by Dragoncraft89** [comment]
Seems like 0. is treated as an enumeration.

If you write 1.0, then this would be formatted as:
`1. 0` (note the space between the point and the zero)

**Comment by tv42** [comment]
Yeah typst seems to think decimal numbers in prose are sentence ending? `$3.141$` and `3\.141` both behave more reasonably.

**Comment by laurmaedje** [comment]
This is just buggy.

**Comment by jgm** [comment]
I believe this issue is fixed and can be closed: see #353 and 
https://github.com/typst/typst/commit/a5c88c19b300c64fafd6a19e3f5f2fe6422a7b67
